packages/framework-provider-aws/test/index.test.ts

```diff
@@ -0,0 +1,69 @@
+import { ProviderLibrary } from '@boostercloud/framework-types'
+import { expect } from '@boostercloud/framework-provider-aws-infrastructure/test/expect'
+import { fake } from 'sinon'
+
+const rewire = require('rewire')
+const providerPackage = rewire('../src/index')
+const fakeInfrastructure = fake.returns({})
+providerPackage.__set__('loadInfrastructurePackage', () => ({
+  Infrastructure: fakeInfrastructure,
+}))
+
+describe('the `framework-provider-aws` package', () => {
+  describe('the `AWSProvider` function', () => {
+    context('with no rockets', () => {
+      const providerLibrary: ProviderLibrary = providerPackage.AWSProvider()
+
+      it('returns a `ProviderLibrary` object', () => {
+        expect(providerLibrary).to.be.an('object')
+        expect(providerLibrary.api).to.be.an('object')
+        expect(providerLibrary.auth).to.be.an('object')
+        expect(providerLibrary.connections).to.be.an('object')
+        expect(providerLibrary.events).to.be.an('object')
+        expect(providerLibrary.graphQL).to.be.an('object')
+        expect(providerLibrary.infrastructure).to.be.a('function')
+        expect(providerLibrary.readModels).to.be.an('object')
+      })
```

Comment:
This test is not needed. As everything is under the Typescript compiler realm, it is impossible to return different types on each of these fields or to return something different than `ProvidersLibrary` from `AWSProvider()`

Reply:
The compiler is great, and you're probably right, but who trusts what happens in the JS runtime realm!!?? :trollface: